defaults/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+ROLE_INDEPENENT_USEFULL_PACKAGES:
+  - htop
+  - ntp
+  - sudo
+  - vim
+  - iftop
+  - iotop
+  - mosh
+  - git
+  - screen
+  - build-essential
+  - apticron
+  - apt-file
+  - etckeeper
+  - links
+  - tree
+  - pwgen
+  - libxml2-dev
+  - libxslt1-dev
+  - python-dev
+  - tmpreaper
```

Comment:
For etckeeper and tmpreaper, just installing it isn't enough - they also needs to be configured. I saw that you included the configuration in another role - which is fine, but then the package dependency must be declared by the role configuring it.